configure

```diff
@@ -17765,7 +17765,53 @@ if test "x$ac_cv_lib_m_pow" = xyes; then :
   LIBS="$LIBS -lm"; EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lm"
 fi
 
-    for ac_func in acos acosf asin asinf atan atanf atan2 atan2f ceil ceilf copysign copysignf cos cosf exp expf fabs fabsf floor floorf trunc truncf fmod fmodf log logf log10 log10f lround lroundf pow powf round roundf scalbn scalbnf sin sinf sqrt sqrtf tan tanf
```

Comment:
The generated configure script is somehow broken? Attaching patch against your current branch: [patch.txt](https://github.com/libsdl-org/SDL/files/8583620/patch.txt)

Reply:
That's weird.  I applied your patch and re-pushed.